content/id/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -0,0 +1,414 @@
+```
+
+<!--
+View detailed information about the Pod:
+-->
+Lihat informasi mendetil tentang Pod:
+
+```shell
+kubectl get pod qos-demo-3 --namespace=qos-example --output=yaml
+```
+
+<!--
+The output shows that Kubernetes gave the Pod a QoS class of BestEffort.
+-->
+Keluaran menampilkan bahwa Kubernetes memberikan kelas QoS BestEffort pada Pod.
+
+```yaml
+spec:
+  containers:
+    ...
+    resources: {}
+  ...
+status:
+  qosClass: BestEffort
+```
+
+<!--
+Delete your Pod:
+-->
+Hapuslah Pod:
+
+```shell
+kubectl delete pod qos-demo-3 --namespace=qos-example
+```
+
+<!--
+## Create a Pod that has two Containers
+
+Here is the configuration file for a Pod that has two Containers. One container specifies a memory
+request of 200 MiB. The other Container does not specify any requests or limits.
+-->
+## Membuat sebuah pod yang memiliki dua Container
+
+Berikut adalah konfigurasi berkas untuk Pod yang memiliki dua Container. Satu Container menentukan permintaan memori sebesar 200MiB. Container yang lain tidak menentukan permintaan atau batasan apapun.
+
+{{< codenew file="pods/qos/qos-pod-4.yaml" >}}
+
+<!--
+Notice that this Pod meets the criteria for QoS class Burstable. That is, it does not meet the
+criteria for QoS class Guaranteed, and one of its Containers has a memory request.
+
+Create the Pod:
+-->
+Perhatikan bahwa Pod ini memenuhi kriteria untuk kelas QoS Burstable. Maksudnya, Container tersebut tidak memenuhi 
+kriteria untuk kelas QoS Guaranteed, dan satu dari Container tersebut memiliki permintaan memori.
+
+Buatlah Pod:
+
+```shell
+kubectl create -f https://k8s.io/examples/pods/qos/qos-pod-4.yaml --namespace=qos-example
+```
+
+<!--
+View detailed information about the Pod:
+-->
+Lihat informasi mendetil tentang Pod:
+
+```shell
+kubectl get pod qos-demo-4 --namespace=qos-example --output=yaml
+```
+
+<!--
+The output shows that Kubernetes gave the Pod a QoS class of Burstable:
+-->
+Keluaran menampilkan bahwa Kubernetes memberikan kelas QoS Burstable pada Pod.
+
+```yaml
+spec:
+  containers:
+    ...
+    name: qos-demo-4-ctr-1
+    resources:
+      requests:
+        memory: 200Mi
+    ...
+    name: qos-demo-4-ctr-2
+    resources: {}
+    ...
+status:
+  qosClass: Burstable
+```
+
+<!--
+Delete your Pod:
+-->
+Hapuslah Pod:
+
+```shell
+kubectl delete pod qos-demo-4 --namespace=qos-example
+```
+
+<!--
+## Clean up
+
+Delete your namespace:
+-->
+## Membersihkan
+
+Hapuslah Namespace:
+
+```shell
+kubectl delete namespace qos-example
+```
+
+## {{% heading "whatsnext" %}}
+
+<!--
+### For app developers
+
+* [Assign Memory Resources to Containers and Pods](/docs/tasks/configure-pod-container/assign-memory-resource/)
+
+* [Assign CPU Resources to Containers and Pods](/docs/tasks/configure-pod-container/assign-cpu-resource/)
+-->
+### Untuk pengembang aplikasi
+
+* [Menetapkan sumber daya memori ke Container dan Pod](/docs/tasks/configure-pod-container/assign-memory-resource/)
```

Comment:
we already have this page translated, change it to /id/docs/tasks/configure-pod-container/assign-memory-resource/